### ISSUE TITLE
[strings] Improve open/closed wording

### ISSUE THREAD

**Issue by matheusgomesms** [organicmaps/organicmaps#7568]
I don't know if this is the correct wording, since EN is not my first language, but I see that OM uses the same string for Open/Closed POIs both in the Place Page and Opening Hours Editor Page. To me, it should be:

Place Page: Open/Closed
Editor: **Opens/Closes** <- suggested change

This is especially true in Romance languages, makes it more correct. Since it's using the same string, I can't properly translate it.

Does it make sense for you guys as well or is it just nit-picking from my side? @organicmaps/translations

(If something like #4212 were implemented probably we shouldn't have this discussion, I believe)

![IMG_2552](https://github.com/organicmaps/organicmaps/assets/86851490/5c67abe3-2afa-49b8-a38a-828ae62e8762)
![IMG_2553](https://github.com/organicmaps/organicmaps/assets/86851490/88a53060-41f6-43b0-a810-7169a055a351)

**Comment by RedAuburn** [comment]
Both types are fine in EN 👍

I guess _technically_ opens/closes is more correct, but i wouldn't bother with it really. The only difference is that opens/closes is framed more as a question, whereas open/closed is more of a statement.

**Comment by biodranik** [comment]
Trying to understand the issue. Shouldn't it be Opens/Closes everywhere?

**Comment by matheusgomesms** [comment]
> Opens/Closes everywhere?

I don't think so. In the Place Page, that selected item is OPEN or CLOSED (just like [signs in the PoIs entrance](https://www.ateomomento.com.br/wp-content/uploads/2015/01/solid-open-closed.png)), but to edit it in opening hours editor, that PoI OPENS at xx hour, and CLOSES at yy hour.

**Comment by matheusgomesms** [comment]
Perhaps in English doesn't make a lot of difference, but for me it makes quite a lot in Portuguese or Spanish (do you feel the same @patepelo?) If this is very nit-picking of my side, let's close this issue!

**Comment by NeatNit** [comment]
To clarify to anyone who's still unsure:

Right now the place is **open/closed**

It **closes/opens** in XX or at XX

(arguably "opens" is somehow short for "will open", i.e. future tense. same for "closes" = "will close")

I agree with OP but if the issue is only with the editor, I don't know how crucial this is. If someone wants to make a pull request, I imagine it will be accepted. (this is just my gut feeling)

**Comment by matheusgomesms** [comment]
Yep, it is not crucial at all.

Just to give a perspective, I came with this issue because I updated a PT string and realized this. So I actually I would like to know if this is interesting to fix. I could try to (someone can give me some pointers), or I can revert my Portuguese edit.

**Comment by patepelo** [comment]
> Perhaps in English doesn't make a lot of difference, but for me it makes quite a lot in Portuguese or Spanish (do you feel the same @patepelo?) If this is very nit-picking of my side, let's close this issue!

I have my OM set up normally in English and hadn't noticed since the difference is very subtle, and it's true that when you translate literally maybe in the destination language the difference is more notable. 

In Spanish (open = abierto; close= cerrado ; while for opens = abre; closes = cierra is technically more correct. I guess you could say that changing it to "opens" and "closes" is a small improvement. 

We could either have the change in the translation (for ES and PT) or directly at the core. I would take @RedAuburn advice as a native English speaker and if EN won't change then you are more than welcomed to improve the translations, and I'd help with ES.

**Comment by NeatNit** [comment]
Looking at the current strings.txt file, these are the current definitions for **closed** (showing only a few languages):

>  [editor_time_close]
>    tags = android,ios
>    en = Closed
>    de = Geschlossen
>    es = Cerrado
>    he = סגור
>    pt = Fecha
>
>  [editor_time_add_closed]
>    tags = android,ios
>    en = Add Non-Business Hours
>    de = Schließzeiten hinzufügen
>    es = Añadir horas de cierre
>    he = הוסף שעות סגירה
>    pt = Inserir horas de encerrrado
>    pt-BR = Inserir horas sem funcionamento
>
>  [editor_hours_closed]
>    tags = android,ios
>    en = Non-Business Hours
>    de = Schließzeiten
>    es = Horas de cierre
>    he = שעות סגירה
>    pt = Horas sem funcionamento
>
>  [closed_now]
>    comment = Place Page opening hours text
>    tags = ios
>    en = Closed now
>    de = Jetzt geschlossen
>    es = Cerrado ahora
>    he = סגור כעת
>    pt = Fechado agora
>    pt-BR = Fechado agora
>
>  [day_off_today]
>    tags = android,ios
>    en = Closed today
>    de = Heute geschlossen
>    es = Cerrado hoy
>    he = סגור היום
>    pt = Fechado hoje
>    pt-BR = Fechado hoje
>
>  [day_off]
>    tags = android,ios
>    en = Closed
>    de = Geschlossen
>    es = Cerrado
>    he = סגור
>    pt = Fechado
>    pt-BR = Fechado
>
>  [closed]
>    tags = android,ios
>    en = Closed
>    de = Geschlossen
>    es = Cerrado
>    he = סגור
>    pt = Fechado
>    pt-BR = Fechado

I can't find out for sure which one is used where (I'm not familiar with the code and it would take too much effort to figure things out), but if the string names are to be believed, [editor_time_close] is the one that needs to be changed (together with [editor_time_open] presumably). So maybe this can be fixed simply by changing those two strings, but without compiling and making sure the change is correct I can't say with any confidence, and I'm not equipped to compile anything right now.

**Comment by biodranik** [comment]
Any help is welcome!
1. Find which strings are used where, on iOS and Android.
1. Check if strings are already split and only translations should be changed.
2. If the same string is shared, then split it and provide better matching translations.

**Comment by matheusgomesms** [comment]
> Looking at the current strings.txt file, these are the current definitions for **closed** (showing only a few languages):
> 
> I can't find out for sure which one is used where (I'm not familiar with the code and it would take too much effort to figure things out), but if the string names are to be believed, [editor_time_close] is the one that needs to be changed (together with [editor_time_open] presumably). So maybe this can be fixed simply by changing those two strings, but without compiling and making sure the change is correct I can't say with any confidence, and I'm not equipped to compile anything right now.

Thanks for the investigation!

I made some investigation as well. First, it seems the problem is only in iOS, since Android uses in the Editor `[editor_time_from]` and `[editor_time_to]`, where in iOS both **Place Page** and **Editor** are obtaining the strings from `[editor_time_open]` and `[editor_time_close`].

I changed Portuguese in #7425 and that's why it got "wrong".

Some solutions:

1 - We could reuse `[editor_time_from]` and `[editor_time_to]` into iOS.

2 - We could create a new string adapting (removing %@) the existing `[opens_at]` and `[closes_at] (so no need to translate something new):

```
[opens_at]
    tags = ios
    en = Opens at %@
    af = Open om %@
    ar = يفتح في %@
    az = Açılır %@
    be = Адкрываецца ў %@
    ca = Obre a les %@
    de = Öffnet um %@
    es = Abre a las %@
    
[closes_at]
    tags = ios
    en = Closes at %@
    af = Sluit om %@
    ar = يغلق في %@
    az = Bağlanır %@
    be = Закрываецца ў %@
    ca = Tanca a les %@
    de = Schließt um %@
    es = Cierra a las %@
```
3 - Another solution is, in Place Page, use the string `[closed]` and create a new one called `[open]`, and fix `[editor_time_open]` and `[editor_time_close`] to be used only in Editor (hence its name, I suppose).

@biodranik what can be done here? Any ideas?

**Comment by biodranik** [comment]
Thanks for the summary. There is some discrepancy between platforms, that should be synced.

1. iOS has opening times displayed on the place page. Android doesn't have it. Is there already an issue for that?
2. Looks like the search results are filled not from the C++ code, but on each platform, right? It would be great to do it in one place someday (if it's possible, of course, e.g. no dependency on some date/time formatting). Maybe create an issue for it?
3. So we have 3 different places: editor, PP and search results.
4. Editor should display "Opens" and "Closes" ("Открывается" and "Закрывается"), or "Opens at"/"Closes at".
5. PP and search should show similar translations ("Open"/"Closed" "Открыто"/"Закрыто"), and also "Closes at ..." and "Closes in ..." depending on time, as @dvdmrtnz implemented it on iOS.

What I'm missing?
